types/react-native/index.d.ts

```diff
@@ -476,6 +476,12 @@ export type PressableStateCallbackType = Readonly<{
     pressed: boolean;
 }>;
 
+export interface PressableAndroidRippleConfig {
+    color?: ColorValue;
```

Comment:
The [upstream typings](https://github.com/facebook/react-native/blob/0.63-stable/Libraries/Components/Pressable/useAndroidRippleForView.js#L29-L31) define these all as ‘maybe’ types, which means they can be `null` as well.

```suggestion
    color?: null | ColorValue;
```